pkg/apis/pipeline/v1alpha1/pipeline_validation_test.go

```diff
@@ -40,20 +41,151 @@ func TestPipelineSpec_Validate_Error(t *testing.T) {
 			},
 		},
 		{
-			name: "invalid constraint tasks",
+			name: "providedby task doesnt exist",
 			fields: fields{
+				Resources: []PipelineDeclaredResource{{
```

Comment:
We're gonna need `builder`s for those :man_dancing:

Reply:
![image](https://user-images.githubusercontent.com/432502/51575334-995cbd00-1e66-11e9-9156-ce7f2916271d.png)

Reply:
i actually did add builders for these! `pipeline_validation_test` isn't using any builders cuz it's doing this odd (where odd = different from all the other tests) thing where it's declaring "fields" and using those to instantiate a Pipeline :thinking: 

ill see what i can do!